translations/de/hints/bryukh.html

```diff
@@ -1,10 +1,10 @@
 <body data-author="bryukh">
 <div class="question-answer">
     <div class="question">
-        I have no idea how to start solving this mission
+        Ich habe keine Ahnung wo ich mit der Lösung anfangen soll.
     </div>
     <div class="answer">
-        You can use a sequence of symbols or digraphs with values.
+        Du kannst eine Folge von Zeichen oder Digrafen mit Werten verwenden.
```

Comment:
The word _Digraf_, in this case relates to given examples like "IX" which is a digraph for the value 9, or "CM" whis is a digraph for the value 900.
I agree that _Digraf_ is a bit confusing, but on the other hand digraphs will be important in other tasks later,
so I think it might be helpful to use this term.
What do you think?